integration-tests/ChatTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using CopilotChat.WebApi.Models.Request;
+using CopilotChat.WebApi.Models.Response;
+using Xunit;
+using static CopilotChat.WebApi.Models.Storage.CopilotChatMessage;
+
+namespace ChatCopilotIntegrationTests;
+
+public class ChatTests : ChatCopilotIntegrationTest
+{
+    [Fact]
+    public async void ChatWithBot()
```

Comment:
Can you make this name more targeted? Maybe something like `ChatMessagePostSucceedsWithValidInput`.